README.md

```diff
@@ -46,16 +46,16 @@ export default Ember.Component.extend({
 You replace it with this:
 
 ```javascript
-import Ember from 'ember'
+import Component from '@ember/component'
 import { action, computed } from 'ember-decorators/object';
 import { service } from 'ember-decorators/service';
 
-export default class MyComponent extends Ember.Component {
```

Comment:
Also, easier to document - ESDoc automatically generates documentation for classes, it would be difficult to do it well for "anonymous" classes